src/parser/tokenizer/yaml/index.js

```diff
@@ -0,0 +1,33 @@
+const yaml = require('js-yaml');
+const { flatten, uniq } = require('lodash');
+
+const toArray = (obj, arr) => {
+  if (obj) {
+    Object.keys(obj).forEach((key) => {
+      const value = obj[key];
+      if (value !== null && typeof value === 'object') {
+        toArray(value, arr)
+      } else if (value) {
+        arr.push(value.toString());
+        }
+    });
+  }
+  return arr;
+};
+
+module.exports = (content, config) => {
+  const result = [];
+  let object;
+
+  if (config) {};
+
+  try {
+    object = yaml.safeLoad(content);
+  } catch (e) {
+      return [];
+    }
+
+  toArray(object, result);
```

Comment:
Is it using a value returned from `toArray()`? It seems to be not assigned to a value.

Reply:
instead of assigning it to a value this pushes results to the array passed as an argument